App/Sagas/NodeLifecycle.ts

```diff
@@ -111,10 +115,14 @@ function * stopNodeAfterDelay (ms: number) {
 }
 
 export function * backgroundTask () {
+  // Check for new photos on every background task event
+  yield put(StorageActions.refreshLocalImagesRequest())
```

Comment:
Same goes for here and below.

Reply:
So doing it this way, we have to rely on the processing of the photo query results to be resilient enough to do the right thing in the case of not being able to interact with the node. Not sure if that's the case or not, but something to keep in mind.